libr/bin/mangling/rust.c

```diff
@@ -0,0 +1,76 @@
+/* radare - LGPL - Copyright 2011-2018 - pancake */
+
+#include <r_bin.h>
+
+#define RS(from, to) (replace_seq ((const char **)&in, &out, (const char *)(from), to))
+
+static bool replace_seq (const char **in, char **out, const char *seq, char value) {
```

Comment:
check with `./sys/clang-format-diff.py`... spaces before `(` etc.

Reply:
sys/indent with clang-format does it wrong.

1) space before ? and : 
2) indents functions with tabs + 2 spaces wtf

Reply:
im fine with the current clang-format output, but we should probably fix that in a separate PR, otherwise reviewing the changes will be a mess

Reply:
r2pm -i r2i